### ISSUE TITLE
AlembicFileFormat targets UV data to non-standard primvars attribute

### ISSUE THREAD

**Issue by nrusch** [PixarAnimationStudios/USD#185]
### Description of Issue

When reading Alembic files, the `UsdAbcAlembicFileFormat` currently applies incoming UV data to the `primvars:uv` attribute on the created prim, which diverges from USD's convention(?) of storing UVs in `primvars:st`.

One of the immediate side-effects of this behavior is that, if the resulting prim is then read into Katana, the UV data ends up in the `geometry.arbitrary.uv` attribute instead of `geometry.arbitrary.st`, and since KtoA (in its current form) is looking for the latter but not the former, subsequent renders will fail.

An argument could certainly be made that KtoA should be able to use UVs data from `geometry.arbitrary.uv` as an alternative, but I still think things should be consistent on the USD side as much as possible.

This may be as simple as updating the target attribute name [here](https://github.com/PixarAnimationStudios/USD/blob/v0.7.4/pxr/usd/plugin/usdAbc/alembicUtil.h#L111).

**Comment by iceprincefounder** [comment]
Yeah,I got a same problem,and I found that the usd files have the attribute `geometry.arbitrary.st` but abc files didn`t have,If you rename `geometry.arbitrary.uv` to `geometry.arbitrary.st` in katana,you can get uv information!This attribute should be renamed I think.

**Comment by jtran56** [comment]
Filed as internal issue #144611.

**Comment by NickWu** [comment]
I was wondering whether this issue is fixed in USD?

**Comment by ainaerco** [comment]
Any update on that?

**Comment by ababak** [comment]
Any news on this issue?

**Comment by spiffmon** [comment]
This is not an issue that we plan to address at Pixar, but we are happy to accept a PR that ensures the unit tests continue to pass.

As to the approach suggested by @nrusch , I would say that unconditionally changing the token value itself _might_ be problematic for some studios... ILM are heavy Alembic/USD users, adding @juliencb2018 for comment.  The alternative, slightly more involved approach would be to add a TfEnvSetting that guards the behavior so that each site can configure the behavior - or if you really needed a more literal translation of the abc file, you could disable to uv -> st translation.  But we’d leave that decision to the community.

**Comment by juliencb2018** [comment]
Thanks for the addition and sorry for the delay in replying.
We (ILM) are currently using an OpScript in Katana, and a PxrManifold2D node for our prman preview shaders. I think our plan was to keep it this way, until we fully move to a usd solution for geometry caching. I expect other studios still using Alembic for their leaf level geom caching did implement similar things and would be affected if the uv attribute was simply renamed to st.
Happy to discuss more!
